.github/workflows/package_api.yml

```diff
@@ -0,0 +1,48 @@
+name: Package API
+on:
+  pull_request_target:
+    types: [labeled, opened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.head.repo.full_name == 'primer/octicons' && contains(github.event.pull_request.labels.*.name, 'Update Public API') }}
```

Comment:
Nitpick: How do you feel about calling the label `api changes approved`? (I don't think we use capital letters in any of our labels)